web_src/js/features/repo-common.js

```diff
@@ -1,6 +1,10 @@
 import $ from 'jquery';
+import Cite from 'citation-js';
```

Comment:
Cite seems to also be in `@citation-js/core`, maybe we can remove this dependency?

```js
const {Cite} = require('@citation-js/core')